specification/kubernetesconfiguration/resource-manager/readme.python.md

```diff
@@ -21,7 +21,6 @@ Generate all API versions currently shipped for this package
 ```yaml $(python)
 multiapi: true
 default-api-version: "2023-05-01"
-clear-output-folder: true
```

Comment:
We don't want to delete it since it can delete outdated files under specific folder.